generator/src/main/java/org/apache/kafka/message/checker/MetadataSchemaCheckerTool.java

```diff
@@ -81,6 +90,23 @@ public static void run(
                         ", and path2: " + path2);
                 break;
             }
+            case "verify-evolution-git": {
+                String filePath = "/metadata/src/main/resources/common/metadata/" + namespace.getString("file");
+                Path rootKafkaDirectory = Paths.get("").toAbsolutePath();
+                while (!rootKafkaDirectory.endsWith("kafka")) {
```

Comment:
this condition depends on the project folder name. What we're actually looking for is the Git folder, so maybe we could change the condition to `while (!new File(rootKafkaDirectory.toFile(), ".git").exists())`, which would allow us to use a different folder name.